stripe-core/src/main/java/com/stripe/android/core/networking/AnalyticsRequestFactory.kt

```diff
@@ -60,7 +60,7 @@ open class AnalyticsRequestFactory(
     }
 
     private fun AnalyticsEvent.params(): Map<String, String> {
-        return mapOf(AnalyticsFields.EVENT to this.toString())
+        return mapOf(AnalyticsFields.EVENT to this.eventName)
```

Comment:
is it possible to cover this in a test?

Reply:
Added a specific test for this: https://github.com/stripe/stripe-android/pull/4660/commits/c76599e308dda0c7c04b31f67c071ef3aef98f34

Also there were a few tests failing on payments that caught this:

```
com.stripe.android.paymentsheet.analytics.DefaultEventReporterTest > onInit() should fire analytics request with expected event value FAILED
    org.mockito.exceptions.verification.junit.ArgumentsAreDifferent at DefaultEventReporterTest.kt:48

com.stripe.android.paymentsheet.analytics.DefaultEventReporterTest > onPaymentSuccess() should fire analytics request with expected event value FAILED
    org.mockito.exceptions.verification.junit.ArgumentsAreDifferent at DefaultEventReporterTest.kt:60

com.stripe.android.paymentsheet.analytics.DefaultEventReporterTest > onSelectPaymentOption() should fire analytics request with expected event value FAILED
    org.mockito.exceptions.verification.junit.ArgumentsAreDifferent at DefaultEventReporterTest.kt:72
```.